### PR TITLE
Ignore IP flags in generate_hashed_packet_to_server in tests/common/dualtor/dual_tor_utils.py

### PR DESCRIPTION
**Pull request by congh-nvidia**
### Description of PR

Summary:
Currently the method generate_hashed_packet_to_server in tests/common/dualtor/dual_tor_utils.py is only used by test case tests/dualtor/test_orchagent_active_tor_downstream.py::test_downstream_ecmp_nexthops. 
The IP flags should be ignored when verifying encapsulated packets, because the "Don't fragment" flag may be set on some platforms, for example the Mellanox platforms, when the packet is encapsulated.

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

#### What is the motivation for this PR?
Ignore IP flags in generate_hashed_packet_to_server to make test test_downstream_ecmp_nexthops pass on Mellanox platforms.
#### How did you do it?
Ignore IP flags in the mask for verifying the encapsulated packets.
#### How did you verify/test it?
Verified by automation, test case test_downstream_ecmp_nexthops passed on Mellanox 4600C dualtor testbed.

### CODE REVIEW

File: tests/common/dualtor/dual_tor_utils.py [hunk 1]
```diff
@@ -872,6 +872,7 @@ def _generate_hashed_ipv4_packet(src_mac, dst_mac, dst_ip, hash_key):
         exp_tunnel_pkt.set_do_not_care_scapy(IP, "id")       # since src and dst changed, ID would change too
         exp_tunnel_pkt.set_do_not_care_scapy(IP, "ttl")      # ttl in outer packet is set to 255
         exp_tunnel_pkt.set_do_not_care_scapy(IP, "chksum")   # checksum would differ as the IP header is not the same
+	exp_tunnel_pkt.set_do_not_care_scapy(IP, "flags")  # "Don't fragment" flag may be set in the outer header
```
Comment (liat-grozovik) [inline tests/common/dualtor/dual_tor_utils.py:875]: not sure if this is the right way or not but all the comments are aligned and the new added one is not. could you please align the indentation of the comment as well?

**Comment by liat-grozovik** [comment]
@congh-nvidia could you please check the pre-commit failure and try to over come them? 
@bingwang-ms FYI

File: tests/common/dualtor/dual_tor_utils.py (hunk 1 above)
Comment (congh-nvidia) [reply tests/common/dualtor/dual_tor_utils.py]: Hi @liat-grozovik，I've aligned the comments and fixed the pre-commit failure. Thanks for your comments!

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**